.buildkite/scripts/steps/ec2-cleanup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export ACCOUNT_KEY_SECRET=$(vault kv get -field=client_email $VAULT_PATH)
+export ACCOUNT_SECRET=$(vault kv get -field=private_key $VAULT_PATH)
+export ACCOUNT_PROJECT_SECRET=$(vault kv get -field=project_id $VAULT_PATH)
```

Comment:
Not sure if these are the correct fields in Vault for EC2? I copied these from the analogous `gce-cleanup.sh` file.  @v1v?

Reply:
I think we need to use a different approach here, and avoid accessing vault secrets at runtime - they are not masked.

The recommended approach is to use https://github.com/elastic/vault-secrets-buildkite-plugin/

However, I think we could use OIDC and avoid the situation of using static secrets:
- https://github.com/elastic/oblt-aws-auth-buildkite-plugin
- plus the OIDC configuration in the specific AWS account